sites/hashdev/src/_pages/blog/7_block-paved-path.mdx

```diff
@@ -301,7 +301,7 @@ Well, you have almost certainly seen and used a block.
 
 _This is a video - click play on the slide_
 
-Most of you will have encountered blocks in an app like [Notion](https://notion.so), which is arguably responsible for the huge popularity boom in blocks.
+Most of you will have encountered blocks in a knowledge-management app like _Notion_, which have (arguably) been responsible for a huge boom in awareness and popularity of blocks.
```

Comment:
```suggestion
Most of you will have encountered blocks in a knowledge-management app like _Notion_, which has (arguably) been responsible for a huge boom in awareness and popularity of blocks.
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/669/commits/143b67cf58dec0722d3516dcdc02be1f47bb0ba9